File: dotnet/Multiplayer/Rooms/Realtime/PlayerInfo.cs
namespace Michitai.Multiplayer.Rooms.Realtime
{
    /// <summary>
    /// Player information for realtime WebSocket connections.
    /// Contains minimal player details for realtime communication.
    /// </summary>
    public class PlayerInfo
    {
        /// <summary>
        /// The unique player ID.
        /// </summary>
        public int player_id { get; set; }

        /// <summary>
        /// The player's display name.
        /// </summary>
        public string player_name { get; set; }

        /// <summary>
        /// The ID of the room the player is in.
        /// </summary>
        public string room_id { get; set; }

        /// <summary>
        /// Whether the player is the host of the room.
        /// </summary>
        public bool is_host { get; set; }
    }
}

File: java/src/main/java/com/michitai/multiplayer/rooms/realtime/PlayerInfo.java
package com.michitai.multiplayer.rooms.realtime;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Player information for realtime WebSocket connections.
 * Contains minimal player details for realtime communication.
 */
public class PlayerInfo {
    @JsonProperty("player_id")
    private int playerId;

    @JsonProperty("player_name")
    private String playerName;

    @JsonProperty("room_id")
    private String roomId;

    @JsonProperty("is_host")
    private boolean isHost;

    public int getPlayerId() {
        return playerId;
    }

    public void setPlayerId(int playerId) {
        this.playerId = playerId;
    }

    public String getPlayerName() {
        return playerName;
    }

    public void setPlayerName(String playerName) {
        this.playerName = playerName;
    }

    public String getRoomId() {
        return roomId;
    }

    public void setRoomId(String roomId) {
        this.roomId = roomId;
    }

    public boolean isHost() {
        return isHost;
    }

    public void setHost(boolean host) {
        isHost = host;
    }
}
